Jenkinsfile

```diff
@@ -5,12 +5,12 @@
 buildPlugin(useContainerAgent: true, configurations: [
   // Test the long-term support end of the compatibility spectrum (i.e., the minimum required
   // Jenkins version).
-  [ platform: 'linux', jdk: '8' ],
+  [ platform: 'linux', jdk: '11' ],
 
   // Test the common case (i.e., a recent LTS release) on both Linux and Windows.
-  [ platform: 'linux', jdk: '11', jenkins: '2.332.1' ],
-  [ platform: 'windows', jdk: '11', jenkins: '2.332.1' ],
+  [ platform: 'linux', jdk: '11', jenkins: '2.361.4' ],
+  [ platform: 'windows', jdk: '11', jenkins: '2.361.4' ],
```

Comment:
Since 2.361.4 is the new minimum version, it is already tested in the earlier line (line 8).  Since 2.361.4 is the new minimum, we don't need to explicitly list the version number in the parameters.  It will be used because it is the default.
```suggestion
  [ platform: 'windows', jdk: '11' ],
```